doc/running_local_cluster.md

```diff
@@ -495,7 +493,7 @@ When you run tests with:
 
 `pytest ... --log-level=debug`
 
-you can see what cardano-cli commands are executed.
+you can see what Cardano-CLI commands are executed.
```

Comment:
It's either 'cardano-cli' or 'Cardano CLI'. I'd suggest just deleting the hyphen